src/h5.cpp

```diff
@@ -14,23 +14,21 @@ void h5d::init_basics() {
   Type = Dataset.datatype();
   DSPTgt = Dataset.dataspace();
   ndims = DSPTgt.rank();
-  LOG(Sev::Debug, "h5d::init_basics");
+  Logger->trace("h5d::init_basics");
   ShapeNow = hdf5::Dimensions(ndims, 0);
   sext = DSPTgt.current_dimensions();
   ShapeMax = DSPTgt.maximum_dimensions();
-  if (log_level >= 9) {
-    for (int i1 = 0; i1 < ndims; ++i1) {
-      LOG(Sev::Debug, "{:20} i: {}  sext: {:21}  ShapeMax: {:21}", Name, i1,
-          sext.at(i1), ShapeMax.at(i1));
+  for (int i1 = 0; i1 < ndims; ++i1) {
+    Logger->trace("{:20} i: {}  sext: {:21}  ShapeMax: {:21}", Name, i1,
+                  sext.at(i1), ShapeMax.at(i1));
+    try {
```

Comment:
Something has gone wrong here, the `try`-`catch` should not be inside the `for` loop. (`for` loop should only contain the `Logger->trace()` call)